test/test_nn_quantized.py

```diff
@@ -32,6 +33,26 @@ def test_relu_api(self):
 
 
 class ModuleAPITest(TestCase):
+    def test_dynamic_linear_api(self):
+        """test API functionality for nn.quantized.DynamicLinear"""
+        batch_size = 1
+        in_features = 2
+        out_features = 2
+        W = torch.rand(out_features, in_features).float()
+        W_scale, W_zp = _calculate_dynamic_qparams(W, torch.qint8)
+        W_q = torch.quantize_linear(W, W_scale, W_zp, torch.qint8)
+        W_pack_col_offset = torch.ops.quantized.fbgemm_linear_prepack(W_q)
+        X = torch.rand(batch_size, in_features).float()
+        B = torch.rand(out_features).float()
+        qlinear = nnq.DynamicLinear(in_features, out_features)
+        qlinear._packed_weight = W_pack_col_offset
+        qlinear.bias = B
+
+        Z_dq = qlinear(X)
+        Z_ref = torch.ops.quantized.fbgemm_linear_dynamic(X, W_pack_col_offset, B)
+
+        self.assertEqual(Z_ref, Z_dq)
+
```

Comment:
A general test that we need to add for all nn.quantized modules is to ensure that scripting works.

Reply:
Good catch! I didn't update this unit test. Will update it.

Reply:
Updated.